### ISSUE TITLE
Case fatality rate

### ISSUE THREAD

**Issue by dirkschumacher** [R4EPI/outbreaks#3]
I guess useful to have a function if CIs are required.

**Comment by aspina7** [comment]
Yeah I think this links nice with a descriptive proportion table (cfr essentially just a proportion).

So just a table with counts, proportion then convenience options to add 95%CIs and also to change denom to be per 10000 or 100000 population.

(while we dont necessarily need these here in outbreak descriptives - could also be used as descriptive table in the survey section before adding weights using survey package) two functions one stone?

**Comment by dirkschumacher** [comment]
But incidence is not a proportion anymore. 
Something like this?

```R
rate_statistics(10, 20, confints = TRUE) # does simple proportions

# computes incidence. E.g. 10 per 10^5 with poisson CIs (or whatever is used here)
rate_statistics(10, population = 100000, confints = TRUE)

```

**Comment by aspina7** [comment]
yeah same same but different. Yep something like that!
